debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/transforms/ExtractNewDocumentStateTest.java

```diff
@@ -198,7 +198,7 @@ public void shouldDropHeartbeatMessages() {
         // when
         SourceRecord transformed = transformation.apply(eventRecord);
 
-        assertThat(transformed).isNull();
+        assertThat(transformed).isSameAs(eventRecord);
```

Comment:
Ah, so this was inconsistent then with the relational SMT then? +1 for aligning it, not sure why they ever were dropped here. Warrants a migration notes entry for sure.